app/code/core/Mage/Eav/Model/Entity/Abstract.php

```diff
@@ -540,6 +548,9 @@ public function loadAllAttributes($object=null)
      */
     public function getSortedAttributes($setId = null)
     {
+        if (null !== $setId && true === isset(self::$_sortedAttributesByAttributeSetId[$setId])) {
```

Comment:
yoda style? `null !== $setId` should be `$setId !== null`.

Reply:
Why? Yoda style is "safer" and should never be considered wrong.

Reply:
Why do you consider it safer? For me its harder/less natural to read.

Reply:
`$var == null` can easily be mistakenly changed to `$var = null`. I'm sure you've done it in your lifetime, especially when not working with a decent IDE.

Reply:
There are pros and cons but I think the deciding factor should be the coding standards. Sad to say I don't know exactly what ours are but I assume we generally follow the standard we inherited from Magento.. Is anyone familiar with this enough to point out where this might be covered? If it isn't covered I would prefer to stick to the existing conventions which may not be consistent but some to be non-yoda style, whatever that's called. :)

Reply:
PSR does not cover this,  wordpress encourages yoda style ( https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions )
Magento does also not cover/enfource it https://github.com/magento/magento-coding-standard/issues/137

Therefore, we should leave this as it is. But if you feel the need to set a rule here, thats a topic for an RFC because of its controversy

Reply:
If it is an topic for an RFC i'd keep current non-yoda-style and discuss to change it later. (Same for short array syntax or return types.)

It is a harder to read and Magento uses "normal" style in most places. I just wanted to mention it ....

Reply:
I do believe it is fair to expect PRs to generally adhere to what is most prevalent in the existing code (non-yoda style and non-explicit comparisons to true/false unless truthy/falsey comparisons are not sufficient), but to @dng-dev credit, it doesn't seem to be covered specifically anywhere.